BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+   
+   ![Connected to BigCommerce](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectedToBigCommerce.png "[Connected to BigCommerce]")
+
+8. Change to your *Actindo Core1* instance and select *Omni-Channel > Settings > Tab CONNECTIONS*.   
+    - The new connection has been created, synchronized, and activated.   
+    - The credentials of the connection have been automatically created.
+    - All related standard attributes, attribute sets, and attribute groups have been created. If required, you can adapt these objects to your needs later using the *DataHub* or *PIM* module.
+    - All relevant APIs settings in *BigCommerce* are automatically created. See the *Store-level API account* in the *BigCommerce* app under *Settings > API*.
+
+   ![New Omni-Channel connection](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OmniChannelConnectionCreated.png "[New Omni-Channel connection]")
+
+   ![Connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectionCredentials.png "[Connection credentials]")
+
+
+
+## Create BigCommerce connection manually
+
+After automatically creating a connection for one *BigCommerce* store, you may need to manually create additional connections for additional stores, if applicable.      
+If you create the connection manually, you have no functional limitations compared to the automatic creation.
+
+
+### Prerequisite
+
+- You have already created a connection automatically.<!---Janik stimmt das, ist das wirklich eine Voraussetzung?-->
+- You have both the *Store hash* and the *Access token* provided by *BigCommerce* at your disposal.
+- In the *Actindo Core1 Platform*, you have permission to create connections in *Omni-Channel*.
+
+
+### Procedure
+
+*Omni-Channel > Settings > CONNECTIONS*
+
+![Omni-Channel connections](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/Connections.png "[Omni-Channel connections]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create connection* view is displayed.
+
+    ![Add connections](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/AddConnection.png "[Add connections]")
+
+2. Enter a name for the connection in the *Name* field.
+
+3. Click the *Driver* drop-down list and select the *BigCommerce* driver.  
+    The *Credentials* section is displayed below the drop-down list. 
+
+4. Enter the hash key of your store in the *Store hash* field and the access token in the
+  *Access token* field. Both data is provided by *BigCommerce*.
+
+5. Click the [SAVE] button.
+
+6. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") [Back] button and check whether the new connection has been synchronized and activated.   
+    - The new connection has been created, synchronized, and activated.   
```

Comment:
In den anderen Connections, die Liste von Verbindungen ist direkt angezeigt (mit oder ohne Bestätigungsmeldung), daher finde ich es unüblich, dass man auf Back drücken (kann auch natürlich sein). Ich finde aber auch merkwürdig, dass man sagt "check whether the connection is activated" und dass man direkt darunter, man behauptet, dass es tatsächlich passiert ist. Vielleicht der erste Satz (Click the Back button and check whether...) umformulieren? Oder 5. und 6. zusammenfügen? (So was? s. unten)

5. Click the [SAVE] button.
   The connection has been created and synchronized... (+ zusäztliche Info) 
      
   Screenshot
   
   Info If necessary, click the Back button...

Reply:
done